src/main/java/com/epam/asset/tracking/api/UserController.java

```diff
@@ -102,4 +103,24 @@ public BusinessProviderDTO getUserData (
 
 		return userData;
 	}
+
+    @GetMapping(value = "/password/{username}", produces = MediaType.APPLICATION_JSON_VALUE)
```

Comment:
If this is a GET endpoint I think it should return something, as you define it will produce JSON in the annotation definition.

Also ->GET /asset/tracking/users/password/{username} is telling me it will return the user's password